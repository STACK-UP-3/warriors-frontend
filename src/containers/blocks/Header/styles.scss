.brand-logo {
	font-family: 'IBM Plex Sans Condensed', sans-serif;
	font-weight: bold;
}

.brand-logo img {
	height: 50px;
	padding-left: 24px;
	padding-right: 24px;
	vertical-align: middle;
}
