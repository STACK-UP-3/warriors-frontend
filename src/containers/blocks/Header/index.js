import React from 'react';
import { Link } from 'react-router-dom';

// Load styles
import './styles.scss';
// Load images
import imgLogo from '../../../../public/images/logo-avatar.png';

export default class Header extends React.Component {
  render() {
    return (
      <header>
        <nav className="bg-primary-color">
          <div className="nav-wrapper">
            <a href="#" className="brand-logo">
              <img className="responsive-img" src={imgLogo} />
              <span>Barefoot Nomad</span>
            </a>
            <ul id="nav-mobile" className="right hide-on-med-and-down">
              <li>
                <Link to="/dashboard">
                  <i className="material-icons">home</i>
                </Link>
              </li>
              <li>
                <Link to="/notifications">
                  <i className="material-icons">notifications</i>
                </Link>
              </li>
              <li>
                <a>
                  <i className="material-icons">account_circle</i>
                </a>
              </li>
            </ul>
          </div>
        </nav>
      </header>
    );
  }
}
