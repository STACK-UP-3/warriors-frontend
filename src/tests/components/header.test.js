import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from '../../components/common/header';

describe('Test <Header/> Component', () => {
  it('Should render Header correctly', () => {
    const header = shallow(<Header />);
    expect(toJson(header)).toMatchSnapshot();
  });
});
