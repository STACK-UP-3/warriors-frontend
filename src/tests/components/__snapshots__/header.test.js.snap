// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Header/> Component Should render Header correctly 1`] = `
<header
  className="navbar-fixed"
>
  <nav
    className="dashboard-header"
  >
    <div
      className="nav-wrapper"
    >
      <a
        className="sidenav-trigger show-on-med-and-down left"
        data-target="slide-out"
        href="#"
      >
        <i
          className="material-icons"
        >
          menu
        </i>
      </a>
      <a
        className="brand-logo white-text"
        href="/"
      >
        <img
          className="responsive-img circle center"
          src="../../assets/logo.png"
        />
      </a>
      <ul
        className="right hide-on-med-and-down"
        id="nav-mobile"
      >
        <li>
          <Link
            to="/dashboard"
          >
            <i
              className="material-icons"
            >
              home
            </i>
          </Link>
        </li>
        <li>
          <Link
            to="/notifications"
          >
            <i
              className="material-icons"
            >
              notifications
            </i>
          </Link>
        </li>
        <li>
          <Link
            to="/account"
          >
            <i
              className="material-icons"
            >
              account_circle
            </i>
          </Link>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
